File: java/src/main/java/io/fortellis/developer/adminapiexample/models/CallbackRequest.java
package io.fortellis.developer.adminapiexample.models;

import lombok.Data;

@Data
public class CallbackRequest {
    final static public String ACCEPTED = "accepted";
    final static public String REJECTED = "rejected";

    private String status;

    public CallbackRequest(String status) {
        this.status = status;
    }
}

File: dotnet/Models/CallbackRequest.cs
namespace AdminAPIImplementation.Models
{
    public class CallbackRequest
    {
        static public string ACCEPTED = "accepted";
        static public string REJECTED = "rejected";
        public string status { get; set; }

        public CallbackRequest(string status)
        {
            this.status = status;
        }
    }
}
